README.md

```diff
@@ -2,6 +2,9 @@
 
 This website is built using [Docusaurus 2](https://v2.docusaurus.io/), a modern static website generator.
 
+## Install yarn
+- For Debian-derived distros (including Ubuntu and Kali), follow [these](https://yarn.bootcss.com/docs/install/#debian-stable) instructions. 
```

Comment:
Please don't introduce Chinese docs for our English site. And we don't need to specify the OS / Distros here.

Maybe we can refer to https://yarnpkg.com/getting-started/install

We can move this section into `Installation`.

Reply:
Reasonable